src/System.Management.Automation/System.Management.Automation.csproj

```diff
@@ -11,12 +11,12 @@
     <!-- the following package(s) are from https://github.com/JamesNK/Newtonsoft.Json -->
     <PackageReference Include="Newtonsoft.Json" Version="10.0.3" />
     <!-- the following package(s) are from https://github.com/dotnet/corefx -->
-    <PackageReference Include="Microsoft.Win32.Registry.AccessControl" Version="4.5.0-rc1-26423-06" />
-    <PackageReference Include="System.IO.FileSystem.AccessControl" Version="4.5.0-rc1-26423-06" />
-    <PackageReference Include="System.Security.AccessControl" Version="4.5.0-rc1-26423-06" />
-    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="4.5.0-rc1-26423-06" />
-    <PackageReference Include="System.Security.Permissions" Version="4.5.0-rc1-26423-06" />
-    <PackageReference Include="System.Text.Encoding.CodePages" Version="4.5.0-rc1-26423-06" />
+    <PackageReference Include="Microsoft.Win32.Registry.AccessControl" Version="4.5.0" />
+    <PackageReference Include="System.IO.FileSystem.AccessControl" Version="4.5.0" />
+    <PackageReference Include="System.Security.AccessControl" Version="4.5.0" />
+    <PackageReference Include="System.Security.Cryptography.Pkcs" Version="4.5.0" />
+    <PackageReference Include="System.Security.Permissions" Version="4.5.0" />
+    <PackageReference Include="System.Text.Encoding.CodePages" Version="4.5.0" />
     <!-- the following package(s) are from the powershell org -->
     <PackageReference Include="Microsoft.Management.Infrastructure" Version="1.0.0-alpha*" />
     <PackageReference Include="PowerShell.Core.Instrumentation" Version="6.0.0-beta*" />
```

Comment:
Have we this APIs in WCP?

Reply:
These are pkgs PowerShell Team publishes

Reply:
Should we migrate to WCP assemplies if we have them on long time?

Reply:
You mean include them in WCP?  Probably not.